Perlang.Common/Expr.cs

```diff
@@ -240,17 +242,21 @@ public override TR Accept<TR>(IVisitor<TR> visitor)
             }
 
             public override string ToString() =>
-                Name.Lexeme;
+                $"#<Identifier {Name.Lexeme}>";
```

Comment:
This was not related to this change per se, but it looked weird when these expressions were being used in "key not found" exceptions. It makes more sense (it looked like it was a string being used as key).